packages/larva-patterns/components/c-title/c-title.twig

```diff
@@ -1,4 +1,8 @@
-<h3 class="c-title {{ modifier_class }} {{ c_title_classes }}">
+{% if c_title_tag == 'span' %}
```

Comment:
To reduce redundance here, you can use the syntax (should be backwards compatible):
```
{% if c_title_tag_text %}
    <{{ c_title_tag_text }} id="{{ c_title_id_attr }}" class="c-title {{ modifier_class }} {{ c_title_classes }}">
{% else %}
    <h3 class="c-title {{ modifier_class }} {{ c_title_classes }}">
{% endif %}

...

{% if c_title_tag_text %}
    </{{ c_title_tag_text }}>
{% else %}
    </h3>
{% endif %}
```
See modules/heading for an example: https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva-patterns/modules/heading/heading.twig

That said, if only these two tags are allowed for a11y reasons, it makes sense to require it via condition - just checking!